case_studies/strong_lensing/convert_catalog.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+from astropy.table import Table
+
+col_names = [
+    "IDENT",
+    "RA",
+    "DEC",
+    "MAG",
+    "BAND",
+    "WEIGHT",
+    "GAL_FILENAME",
+    "PSF_FILENAME",
+    "GAL_HDU",
+    "PSF_HDU",
+    "PIXEL_SCALE",
+    "NOISE_MEAN",
+    "NOISE_VARIANCE",
+    "NOISE_FILENAME",
+    "stamp_flux",
+    "LENSED",
+    "X",
+    "Y",
+    "N1",
+    "HALF_RAD",
+    "FLUX1",
+    "N2",
+    "SCALE_RAD",
+    "FLUX2",
+    "Q",
+    "BETA",
+    "THETA",
+    "X_LENSE",
+    "Y_LENSE",
+    "E1",
+    "E2",
+]
```

Comment:
better to write these as a dict: column_name -> col_dtype

Reply:
`d.keys()` and `d.values()` will be in the same order

Reply:
Changed